provider/cluster/kube/builder_test.go

```diff
@@ -44,14 +52,13 @@ func TestNetworkPolicies(t *testing.T) {
 	// disabled
 	netPolicies, err := np.create()
 	assert.NoError(t, err)
-	assert.Len(t, netPolicies, 0)
```

Comment:
the case where it's turned on is below, [where it says `//enabled`](https://github.com/ovrclk/akash/pull/978/files#diff-02f0b392eb9b3b671f44c006bbd18ec3a5ae6093c611bc7b7b7dee2dc8390594R56-R61)